Libraries/Drei/lisp-syntax.lisp

```diff
@@ -81,7 +81,7 @@ syntax should be run.")
   (:name "Lisp")
   (:pathname-types "lisp" "lsp" "cl")
   (:command-table lisp-table)
-  (:default-initargs :initial-state |initial-state |))
+  (:default-initargs :initial-state '|initial-state |))
 
```

Comment:
I'm reverting this change, it seems that `|initial-state |` is a defined parameter, not meant as a symbol:

```
(define-parser-state |initial-state | (form-may-follow) ())
```

this change breaks the listener at least.